packages/cli/src/databases/migrations/mysqldb/1588157391238-InitialMigration.ts

```diff
@@ -1,58 +1,45 @@
-import { MigrationInterface, QueryRunner } from 'typeorm';
-
-import config from '@/config';
+import type { MigrationContext, MigrationInterface } from '@db/types';
 
 export class InitialMigration1588157391238 implements MigrationInterface {
-	name = 'InitialMigration1588157391238';
-
-	async up(queryRunner: QueryRunner): Promise<void> {
-		const tablePrefix = config.getEnv('database.tablePrefix');
-
+	async up({ queryRunner, tablePrefix }: MigrationContext) {
 		await queryRunner.query(
 			'CREATE TABLE IF NOT EXISTS `' +
 				tablePrefix +
 				'credentials_entity` (`id` int NOT NULL AUTO_INCREMENT, `name` varchar(128) NOT NULL, `data` text NOT NULL, `type` varchar(32) NOT NULL, `nodesAccess` json NOT NULL, `createdAt` datetime NOT NULL, `updatedAt` datetime NOT NULL, INDEX `IDX_' +
 				tablePrefix +
 				'07fde106c0b471d8cc80a64fc8` (`type`), PRIMARY KEY (`id`)) ENGINE=InnoDB',
-			undefined,
 		);
 		await queryRunner.query(
 			'CREATE TABLE IF NOT EXISTS `' +
 				tablePrefix +
 				'execution_entity` (`id` int NOT NULL AUTO_INCREMENT, `data` text NOT NULL, `finished` tinyint NOT NULL, `mode` varchar(255) NOT NULL, `retryOf` varchar(255) NULL, `retrySuccessId` varchar(255) NULL, `startedAt` datetime NOT NULL, `stoppedAt` datetime NOT NULL, `workflowData` json NOT NULL, `workflowId` varchar(255) NULL, INDEX `IDX_' +
 				tablePrefix +
 				'c4d999a5e90784e8caccf5589d` (`workflowId`), PRIMARY KEY (`id`)) ENGINE=InnoDB',
-			undefined,
 		);
 		await queryRunner.query(
 			'CREATE TABLE IF NOT EXISTS`' +
 				tablePrefix +
 				'workflow_entity` (`id` int NOT NULL AUTO_INCREMENT, `name` varchar(128) NOT NULL, `active` tinyint NOT NULL, `nodes` json NOT NULL, `connections` json NOT NULL, `createdAt` datetime NOT NULL, `updatedAt` datetime NOT NULL, `settings` json NULL, `staticData` json NULL, PRIMARY KEY (`id`)) ENGINE=InnoDB',
-			undefined,
 		);
 	}
 
-	async down(queryRunner: QueryRunner): Promise<void> {
-		const tablePrefix = config.getEnv('database.tablePrefix');
-
-		await queryRunner.query('DROP TABLE `' + tablePrefix + 'workflow_entity`', undefined);
+	async down({ queryRunner, tablePrefix }: MigrationContext) {
+		await queryRunner.query('DROP TABLE `' + tablePrefix + 'workflow_entity`');
```

Comment:
why not use template strings?

Reply:
@edumaciel10 We are not looking for community reviews on these PRs.
**Please use regular comments when interacting on PRs here**. 
All PR review comments by non team members are going to be marked as Spam/Off-topic.

Reply:
ok, sorry

Reply:
have some problem if I put emoji +1 into revised prs?

Reply:
That said, we'll likely enable eslint `prefer-template` across the repo at some point, and then auto-fix all such string concatenation.

Reply:
no issue with that at all. please use regular comments to share your concerns and your feedback. **we very much appreciate that**. 
But please refrain from reviewing PRs in general unless you are a maintainer.